roles/preflight-checks/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: check ceph installed
+  command: ceph --version
+  register: result_ceph_installed
+  failed_when: false
+  delegate_to: "{{ groups['ceph_monitors'][0] }}"
+  when: ceph.enabled | default('False') | bool
+
+- include: ceph.yml
+  when:
+    - result_ceph_installed is defined and result_ceph_installed.rc == 0
+    - not skip_prechecks_all | bool
```

Comment:
remove conditional variables to allow skipping the task. We should just have tag in site.yml for the overall play and let one skip preflight play altogether if they choose to. In production one should never skip it.